[Review] Docs and test fixes

nabobalis opened the pull request:
What have you made me do?!

As it stands: online tests pass on my machine, all doc code moved to /docs instead of doc/source, updated requirements for master and added more categories 

There are also misc changes such as I deleted pytest.ini (this i moved into setup.cfg) and tox.ini. These might have broken but the commands do seem to be working on my machine.

Review pretty please!

(I have no idea why those labels were added to my PR)

pep8speaks commented:
Hello @nabobalis! Thanks for submitting the PR.

 - In the file [`docs/conf.py`](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py), following are the PEP8 issues :

> [Line 52:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L52): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 53:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L53): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 54:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L54): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 58:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L58): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 64:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L64): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 79:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L79): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 183:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L183): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 204:67](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L204): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 204:101](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L204): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (116 > 100 characters)
> [Line 205:47](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L205): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 205:101](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L205): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (106 > 100 characters)
> [Line 206:56](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L206): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 207:61](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L207): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 207:101](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/docs/conf.py#L207): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (102 > 100 characters)

 - In the file [`sunpy/database/tests/test_attrs.py`](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py), following are the PEP8 issues :

> [Line 412:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L412): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 413:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L413): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 414:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L414): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 415:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L415): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 416:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L416): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 417:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L417): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 419:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L419): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 420:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L420): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 421:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L421): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 422:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L422): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 423:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L423): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 424:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L424): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 426:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L426): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 443:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L443): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 444:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L444): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 445:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L445): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 446:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L446): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 447:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L447): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 448:13](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_attrs.py#L448): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent

 - In the file [`sunpy/database/tests/test_database.py`](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py), following are the PEP8 issues :

> [Line 476:31](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L476): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 512:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L512): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 519:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L519): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 828:101](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L828): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (110 > 100 characters)
> [Line 829:101](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L829): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (109 > 100 characters)
> [Line 860:68](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L860): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 860:70](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L860): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 861:67](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L861): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 861:69](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_database.py#L861): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals

 - In the file [`sunpy/database/tests/test_tables.py`](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py), following are the PEP8 issues :

> [Line 31:101](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L31): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (104 > 100 characters)
> [Line 102:89](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L102): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 102:90](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L102): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 103:63](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L103): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 103:64](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L103): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 104:61](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L104): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 104:62](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L104): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 105:67](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L105): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 105:68](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L105): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 304:89](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L304): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 304:90](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L304): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 305:64](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L305): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 305:65](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L305): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 306:62](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L306): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 306:63](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L306): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 307:67](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L307): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 307:68](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L307): [E262](https://duckduckgo.com/?q=pep8%20E262) inline comment should start with '# '
> [Line 322:29](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L322): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 326:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L326): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 327:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L327): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 328:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L328): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 329:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L329): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 330:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L330): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 331:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L331): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 332:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L332): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 333:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L333): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 334:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L334): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 335:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L335): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 336:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L336): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 337:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L337): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 338:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L338): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 339:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L339): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 340:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L340): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 341:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L341): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 342:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L342): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 343:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L343): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 344:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L344): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 345:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L345): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 346:1](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/database/tests/test_tables.py#L346): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented

 - In the file [`sunpy/tests/setup_command.py`](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/tests/setup_command.py), following are the PEP8 issues :

> [Line 62:9](https://github.com/sunpy/sunpy/blob/d43749bf01c98717b803e77d195977b1e366b9be/sunpy/tests/setup_command.py#L62): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '

Cadair commented on requirements/base.txt:
```diff
@@ -1,7 +1,7 @@
 scipy
-pandas>=0.12.0
-matplotlib>=1.1
+pandas>=0.20.0
```
Do we actually need to require these? They are supposed to be the minimum possible versions.

nabobalis replied:
Probably not. That can be changed.

Cadair commented on setup.cfg:
```diff
@@ -26,7 +36,7 @@ edit_on_github = True
 
 [pycodestyle]
-max_line_length = 100
+max_line_length = 120
```
Why? Pep8 allows up to 100.

nabobalis replied:
120 is a better number.

Cadair replied:
I would rather stick to 100.

nabobalis replied:
Fine.

nabobalis commented:
To clarify what I have done here: 

* Fixed the online tests. I have changed the values to match the server but I have left the old values commented out in the files for future reference. 
* Moved doc/source to docs and changed the other files that relied on the old doc location
* I changed up the requirements to be more granular. Now a base, extra, dev and doc files
* Deleted tox.ini since we do not use it anywhere and added pytest.ini to setup.cfg

dpshelio commented on docs/conf.py:
```diff
@@ -203,9 +203,9 @@
+        'examples_dirs': '..{}examples'.format(os.sep), # path to the examples scripts
         'gallery_dirs': 'generated{}gallery'.format(os.sep), # path to save gallery generated examples
-        'default_thumb_file': '..{}logo{}sunpy_icon_128x128.png'.format(os.sep, os.sep),
+        'default_thumb_file': '.{}logo{}sunpy_icon_128x128.png'.format(os.sep, os.sep),
```
You don need two `os.sep` in format. Though I don't really like to use `os.sep` in this way... I rather use `os.path.join`... but I imagine is the same...

nabobalis replied:
Changed just for you

dpshelio commented on sunpy/database/tests/test_attrs.py:
```diff
@@ -406,22 +406,22 @@ def test_walker_create_fitsheader_inverted(session):
-            wavemax=7.293187818826002e-05)]
-
+            instrument=u'RHESSI', size=-1.0, wavemin=0.4132806579880238,
+            wavemax=7.293188082141598e-05)]
```
@ehsteve - are these changes something that could occur in the future again? do we know whether this is for some recalibration?

dpshelio commented on sunpy/database/tests/test_tables.py:
```diff
@@ -99,10 +99,10 @@ def test_entry_from_qr_block(query_result):
+        fileid='/archive/soho/private/data/processed/eit/lz/2001/01/efz20010101.000042', #010014
+        observation_time_start=datetime(2001, 1, 1, 0, 0, 42), #(2001, 1, 1, 1, 0, 14)
+        observation_time_end=datetime(2001, 1, 1, 0, 0, 54), #datetime(2001, 1, 1, 1, 0, 21)
+        instrument='EIT', size=2059.0, wavemin=19.5, wavemax=19.5) #wavemin=17.1, wavemax=17.1
```
I'm not too convinced on writing coments to keep an eye on what was in the past... Mostly in the current way. If someone looks at this file without knowing that these comments are an old result given by the server... then it's very confusing. Could you write a line below the changes a bit more explanatory? Or not comment at all and add these changes in an issue where we can always point to and reopen when necessary, so it's easier to track when the differences were spotted and inform the data providers (eventually finding out where the bug is happening).

Cadair replied:
maybe put the sha hash of the previous commit in the comment?!

nabobalis replied:
Yeah that's a fair point. I did it for my sake when I was editing the files.

nabobalis replied:
I have chosen to delete the comments and dump it into an issue.

dpshelio commented on sunpy/database/tests/test_tables.py:
```diff
@@ -298,12 +298,13 @@ def test_entries_from_query_result(query_result):
     entries = list(entries_from_query_result(query_result))
     assert len(entries) == 122
     snd_entry = entries[1]
+    print(snd_entry)
```
this `print` shouldn't be here

dpshelio commented on sunpy/database/tests/test_tables.py:
```diff
@@ -318,29 +319,31 @@ def test_entry_from_query_results_with_none_wave(qr_with_none_waves):
     entries = list(entries_from_query_result(qr_with_none_waves, 'nm'))
-    assert len(entries) == 7
+    assert len(entries) == 4 # Was 7 Bug or Server issue?
+    print(entries)
```
neither this one

dpshelio commented on sunpy/database/tests/test_tables.py:
```diff
@@ -318,29 +319,31 @@ def test_entry_from_query_results_with_none_wave(qr_with_none_waves):
+#            observation_time_start=datetime(1996, 4, 11, 0, 0, 0),
+#            observation_time_end=datetime(2014, 3, 30, 23, 59, 0),
+#            instrument='VIRGO', size=32652.0, wavemin=None,
+#            wavemax=None),
```
This should go also to the issue I mentioned above, not as commented code.
